server_admin/topics/identity-broker/saml.adoc

```diff
@@ -77,6 +77,26 @@ This link is an XML document describing metadata about the IDP.
 
 You can also import all this configuration data by providing a URL or XML file that points to the entity descriptor of the external SAML IDP you want to connect to.
 
+[[_identity_broker_saml_requested_authncontext]]
+==== Requesting specific AuthnContexts
+Some Identity Providers let the clients specify particular constraints on the authentication method used to verify the user identity (e.g. asking for MFA, Kerberos authentication, security requirements, etc.). These are specified using particular AuthnContext criteria. A client can ask for one or more criteria and also specify how the Identity Provider should match the requested AuthnContext - exactly, or by satisfying same-or-better equivalents.
```

Comment:
Please use "such as" in place of "e.g."  and use "and so on" in place of "etc."

Please always follow "this" or "these" with a noun such as "These constraints..."